### PR TITLE
Fixes #766 Show notifications when network status of device changes

### LINKED ISSUE #766: Show notifications when network status of device changes.
**Summary:** 

Whenever the user goes offline or online,a snackBar should give notification to the User.

**Screen-shots:** 
whatsapp image 2018-02-16 at 00 06 57-2
whatsapp image 2018-02-16 at 00 06 57

### PR DESCRIPTION
**Pull request by Siddharth-Singhs**
## Description

Whenever the user goes offline or online,a snackBar should give notification to the User.

## Related issues and discussion
#766 
 
 ## Screen-shots, if any
whatsapp image 2018-02-20 at 19 18 22-2
whatsapp image 2018-02-20 at 19 18 22

 
 ## Checklist
 
 Please make sure these boxes are checked before submitting your pull request - thanks!
 
 
 
 - [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 
 - [x] If you have multiple commits please combine them into one commit by squashing them.
 
 - [x] Read and understood the contribution guidelines .

### CODE REVIEW

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java [hunk 1]
```diff
@@ -565,4 +587,63 @@ public void moveToBarcodeEntry() {
...
+            String status = utils.getConnectivityStatusString(context);
+            setSnackbarMessage(status,false);
+        }
+    };
+    private void setSnackbarMessage(String status,boolean showBar) {
+        String internetStatus="";
+        if(status.equalsIgnoreCase("Wifi enabled")||status.equalsIgnoreCase("Mobile data enabled")){
+            internetStatus=getResources().getString(R.string.offline_network_connected);
+        }else {
+            internetStatus=getResources().getString(R.string.offline_network_disconnected);
```
Comment (policorn) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java]: `offline_network_disconnected` is not defined in strings.xml

**Comment by policorn** [comment]
Hi @teolemon, this one can be merged

**Comment by policorn** [comment]
Hi @Siddharth-Singhs why did you pushed those commits to this PR?

**Comment by Siddharth-Singhs** [comment]
This Fixes #813 Add Intro Screens update the Strings.xml which caused branch conflicts.
So I have to send new commit to remove Branch Conflicts.

**Comment by policorn** [comment]
Hi @Siddharth-Singhs we had a issue in master (already resolved) fixed in a different PR; can you please undo/review your commits to fix the conflict so we can merge your PR?

**Comment by Karljoones** [comment]
@Siddharth-Singhs @policorn I think that we should close this pull request, the app has gotten to the point that it checks connectivity at times when the user needs it rather than just keeping them updated with it. The user should only be informed of the status of the network connection at the time of doing an action that needs a connection. I don't think that we really need this feature! I won't close it till I hear back from you though. :)

**Comment by policorn** [comment]
I agree with @Karljoones and I also think we should abandon this one

**Comment by Karljoones** [comment]
Closing pull request, sorry @Siddharth-Singhs and thank you for all your work!

**Pull request closed without merging**